infra/main.bicep

```diff
@@ -53,6 +53,27 @@ param machineLearningName string = '${environmentName}-aml-${resourceToken}'
 @description('Id of the user or app to assign application roles')
 param principalId string = ''
 
+@description('Flag to deploy resources in isolated network. Manual setup is required to access the deployed resources.')
+param DeployResourcesWithIsolatedNetwork bool
+
+@description('Name of the virtual network')
+param VirtualNetworkName string = '${environmentName}vnet'
+
+@description('Address space for the virtual network')
+param VnetAddressSpace string = ''
+
+@description('Subnet name for the proxy server subnet')
+param ProxySubnetName string = 'AzureBastionSubnet'
+
+@description('Address space for the proxy server subnet')
+param ProxySubnetAddressSpace string = ''
+
+@description('Subnet name for other azure resources')
```

Comment:
does this need to be a param?
We don't set it, so why not leave it as a variable, rather than a parameter?